R/styles.R

```diff
@@ -22,3 +28,13 @@ style <- function(..., sep = "") {
 
   paste(unlist(x), collapse = sep)
 }
+
+red <- rcmdcheck_color(crayon::red)
```

Comment:
To be safest maybe we should do these onLoad? Though I am not sure it matters much...

Reply:
Yeah, I added that.